.github/workflows/hq-smoke-tests.yml

```diff
@@ -108,10 +109,16 @@ jobs:
                 .replace(/{{runNumber}}/g, context.runNumber)
                 .replace(/{{environment}}/g, CC_ENV)
 
+            let receivers = qa@dimagi.com, sameena.shaik@fissionlabs.com
+            if (EVENT_NAME !== "pull_request") {
```

Comment:
the event name is already available via `context.eventName` (see similar if statement on line 97)